cmd/experimental/kjobctl/apis/v1alpha1/volumebundle_types.go

```diff
@@ -17,11 +17,35 @@ limitations under the License.
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type MountPoint struct {
+	// Inlines Container’s volume mounts.
+	corev1.VolumeMount `json:",inline"`
+
+	// Environment variables pointing to locations on the mount.
+	// These values will be copied to pods/containers environment variables.
+	// +optional
+	Envs []string `json:"envs,omitempty"`
```

Comment:
should match "[A-Za-z_][A-Za-z0-9_]*"

Reply:
The type is problematic here. If we go with multiple environment variables then they need to be pairs - variable name & value/path

Reply:
Moved envVars to VolumeBundleSpec as discussed in the design document.